rpc/gnmi/gnmi.proto

```diff
@@ -129,6 +155,23 @@ message Error {
   google.protobuf.Any data = 3;   // Optional additional information.
 }
 
+
+// Decimal64 is used to encode floating-point decimal number. The value
+// expressed as a set of digits with the precision specifying the number
+// of digits following the decimal point in the digit set.
+message Decimal64 {
+  uint64 digits = 1;      // Set of digits.
+  uint32 precision = 2;    // Number of digits following the decimal point.
+}
+
+// ScalarArray is used to encode a mixed-type array of values.
+message ScalarArray {
+  // The set of elements within the array. Each TypedValue message should
+  // specify only elements that have a field identifier of 1-7 (i.e., the
+  // values are scalar values).
+  repeated TypedValue element = 1;
```

Comment:
given the restriction on the types in the scalar array, does it make sense to list out the types here, or maybe split TypedValue message into 'simple' and 'complex' type messages ?